src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -2404,7 +2404,9 @@ private function createSignalableApplication(Command $command, ?EventDispatcherI
         if ($dispatcher) {
             $application->setDispatcher($dispatcher);
         }
-        $application->add(new LazyCommand($command::getDefaultName(), [], '', false, fn () => $command, true));
+        /** @var AsCommand $attribute */
+        $attribute = ((new \ReflectionClass($command))->getAttributes(AsCommand::class)[0] ?? null)?->newInstance();
+        $application->add(new LazyCommand($attribute->name, [], '', false, fn () => $command, true));
```

Comment:
You know the class has the attribute, and you assume `$attribute` is not null when calling `$attribute->name`.
```suggestion
        /** @var AsCommand $attribute */
        $attribute = ((new \ReflectionClass($command))->getAttributes(AsCommand::class)[0])->newInstance();
        $application->add(new LazyCommand($attribute->name, [], '', false, fn () => $command, true));
```